Mathlib/RingTheory/MvPolynomial/NewtonIdentities.lean

```diff
@@ -153,6 +153,7 @@ private theorem sum_filter_pairs_eq_sum_powersetCard_mem_filter_antidiagonal_sum
   have : card p.fst ≤ k := by apply le_of_lt; aesop
   aesop
 
+set_option maxHeartbeats 400000 in
```

Comment:
Do you understand why this is now needed?

Reply:
Nothing deep, it seems. It takes 186494 heartbeats on master and one of my commits made it go up closer to the max of 200k.

Since it seemed a bit unstable, I've now golfed it down to 13k heartbeats.